contracts/SecurityModule.sol

```diff
@@ -9,47 +9,43 @@ import "./IModuleRegistry.sol";
 
 contract SecurityModule is BaseModule, Initializable {
 
-    uint public locked_security_period;
-    uint public recovery_security_period;
 
     struct Recovery {
         uint activeAt; // timestamp for activation of escape mode, 0 otherwise
         address recovery;
     }
     mapping (address => Recovery) public recoveries;
 
-    struct SignerInfo {
+    struct SignerConfInfo {
         address[] signers;
         bool exist;
+        uint locked_security_period;
+        uint recovery_security_period;
     }
 
-    mapping (address => SignerInfo) public signerInfos;
+    mapping (address => SignerConfInfo) public signerConfInfos;
     constructor() {}
 
-    function initialize(
-        IModuleRegistry _registry,
-        uint _locked_security_period,
-        uint _recovery_security_period
-    ) public initializer {
+    function initialize(IModuleRegistry _registry) public initializer {
         registry = _registry;
-        locked_security_period = _locked_security_period;
-        recovery_security_period = _recovery_security_period;
     }
 
     function init(address _wallet, bytes memory data)  public override onlyWallet(_wallet) {
         require(!isRegisteredWallet(_wallet), "SM: should not add same module to wallet twice");
-        require(!signerInfos[_wallet].exist, "SM: wallet exists in signerInfos");
+        require(!signerConfInfos[_wallet].exist, "SM: wallet exists in signerConfInfos");
 
         addWallet(_wallet);
         // decode signer info from data
-        (address[] memory signers) = abi.decode(data, (address[]));
-        SignerInfo storage signerInfo = signerInfos[_wallet];
+        (address[] memory signers, uint lockedSecurityPeriod, uint recoverySecurityPeriod) = abi.decode(data, (address[], uint, uint));
```

Comment:
In most case, the user does not care about those two periods, so  we can set  up  an initial value like before we do, as a result,  it saves some gas for the wallet creators.